src/together/types/completions.py

```diff
@@ -61,7 +61,7 @@ def verify_parameters(self) -> Self:
 class CompletionChoicesData(BaseModel):
     index: int
     logprobs: LogprobsPart | None = None
-    finish_reason: FinishReason
+    finish_reason: FinishReason | None = None
```

Comment:
Do we not use Optional types here?

Reply:
This is the recommended way Python>3.9. See [PEP 604](https://peps.python.org/pep-0604/). And [related discussion](https://discuss.python.org/t/clarification-for-pep-604-is-foo-int-none-to-replace-all-use-of-foo-optional-int/26945) on this topic.